### PR TITLE
[Bug fix] Logs always relative to starting directory

### PR DESCRIPTION
**Pull request by alexandraorth**
This is a bug fix for #15307, where the logs were always being placed relative to the directory where grakn was starting. 

Now, if a relative log dir is specified in the properties file, it will be relative to the system variable grakn.dir. Otherwise the absolute value is used.

### CODE REVIEW

File: conf/main/logback.xml [hunk 1]
```diff
@@ -21,32 +21,46 @@
     <property name="graknLogFileName" value="grakn.log"/>
     <property name="graknPPLogFileName" value="grakn-postprocessing.log"/>
 
+    <!--Configure a properties file (located on the classpath) from which values can be referenced-->
     <property resource="grakn.properties"/>
 
+    <!--Extract the directory of the grakn logs from the properties file. If the path is relative it
+    will be appended to the directory containing grakn (system property grakn.dir). Otherwise the absolute
+    path is used. -->
+    <if condition='property("log.dirs").startsWith("/")'>
```
Comment (fppt) [inline conf/main/logback.xml]: Ewwwwwwwwwww . . . Oh god. Conditionals in xml . . . ugh.

**Comment by fppt** [comment]
@alexandraorth merge this into master and reference here please

File: conf/main/logback.xml (hunk 1 above)
Comment (aelred) [inline conf/main/logback.xml]: wow

Comment (aelred) [inline conf/main/logback.xml]: there is no god

Comment (alexandraorth) [inline conf/main/logback.xml]: I know :( Can you guys think of another way??

Comment (fppt) [inline conf/main/logback.xml]: No, but that does not mean I won't judge you for putting conditional XML statements in

Comment (alexandraorth) [inline conf/main/logback.xml]: </3

**Comment by alexandraorth** [comment]
#1758

**Comment by kasper-piskorski** [comment]
merge here: https://github.com/graknlabs/grakn/pull/1760

**Pull request merged**